config-linux.md

```diff
@@ -494,8 +494,8 @@ For more information, see [the man page](http://man7.org/linux/man-pages/man8/sy
 
 Seccomp provides application sandboxing mechanism in the Linux kernel.
 Seccomp configuration allows one to configure actions to take for matched syscalls and furthermore also allows matching on values passed as arguments to syscalls.
-For more information about Seccomp, see [Seccomp kernel documentation](https://www.kernel.org/doc/Documentation/prctl/seccomp_filter.txt)
-The actions, architectures, and operators are strings that match the definitions in seccomp.h from [libseccomp](https://github.com/seccomp/libseccomp) and are translated to corresponding values.
+For more information about seccomp, see [seccomp][seccomp] kernel documentation.
```

Comment:
When the link text matches the reference, you can skip the target (e.g. `[seccomp][]`).  See the [docs for the *implicit link name* shortcut][1].  Taking advantage of that makes the line here a bit easier to read, since it removes the distraction of an explicit reference name.

[1]: http://daringfireball.net/projects/markdown/syntax#link